src/Traits/Configurable.php

```diff
@@ -106,7 +113,7 @@ private function mapScopes()
 			}
 		}
 
-		return $mappedScopes;
+		return  array_merge( $mappedScopes , $this->_config[ 'gmail.additional_scopes' ] );
```

Comment:
Following from the config file `additional_scopes`, I'm already doing this but with the regular scopes

Reply:
https://github.com/dacastro4/laravel-gmail/pull/65#issuecomment-486811868